packages/ra-auth-cognito-language-english/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import type { CognitoTranslationMessages } from 'ra-auth-cognito';
+
+export const raAuthCognitoEnglishMessages: CognitoTranslationMessages = {
+    ra: {
+        auth: {
+            cognito: {
+                mfa: {
+                    totp: {
+                        association_helper_text:
+                            'Enter the verification code generated by your authenticator app to complete the association.',
+                        association_label: 'Verification code',
+                        association_required:
+                            'Two steps authentication is required, please associate an authenticator to your account by scanning the QR code or entering the secret key.',
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** As a user, I don't really know what to do by reading this sentence. What do you think about:
‏
```suggestion
                            'Two-step authentication required. If you haven't, please download a TOTP authenticator to your phone. Then associate it with your account by scanning the QR code below and enter the secret code generated by your application.',
```

Reply:
Example of the msg used by Discord:

> You can configure two-factor authentication (2FA) using a TOTP app on mobile or desktop or via text message. After you have configured 2FA using a TOTP app or via text message, you can then also add security keys as alternate 2FA methods.

Reply:
The example has informations we don't handle like text message or security keys.
The message used by discord is only `You can configure two-factor authentication (2FA) using a TOTP app on mobile or desktop.`